app/js/components/pages/App.jsx

```diff
@@ -14,6 +14,7 @@ const App = ({ connected }) => (
                 <Directory />
             </div>
             <PluginsList />
+            <br></br>
```

Comment:
`<br>` is a `void` element, which means it can't have content, so the above (`<br></br>`) is "technically" illegal and should be either `<br>` or `<br />`. I would go with the latter.